packages/theme/src/lib/legacy-theme.ts

```diff
@@ -1024,6 +1024,14 @@ export const legacyTheme = {
         },
       },
     },
+    MuiPopover: {
+      defaultProps: {
+        elevation: 0,
+        PaperProps: {
+          variant: 'elevation',
```

Comment:
Based on reactstrap and the availity-react feedback component this should probably be outlined instead